lib/atlas/scaler/graph_persistor.rb

```diff
@@ -0,0 +1,26 @@
+module Atlas
+  class Scaler::GraphPersistor
+    def initialize(dataset, path)
+      @dataset = dataset
+      @graph   = GraphBuilder.build
+      @path    = path
+    end
+
+    def persist_graph
```

Comment:
`execute!` doesn't state what it does. I'm not a fan of those method names personally. It's the same as `do_something!`, `fire!`, `work!`, it doesn't tell you anything. I would rather change it to `persist!` since than it's actually clear to me.